view/next-project/src/components/sponsoractivities/OpenPaymentDayModalButton.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { useState } from 'react';
+
+import { PrimaryButton } from '../common';
+import PaymentDayModal from './PaymentDayModal';
+import { SponsorActivityView } from '@/type/common';
+
+interface Props {
+  children?: React.ReactNode;
+  sponsorActivitiesViewItem: SponsorActivityView;
+  setIsOpen: (isOpen: boolean) => void;
+  isDisabled?: boolean;
```

Comment:
```suggestion
  isDisabled: boolean;
```
undefinedを許容しなくて動くと思います